packages/Confirmation/src/Confirmation.js

```diff
@@ -104,7 +107,7 @@ const Confirmation = props => {
   const I18n = useI18n();
 
   const popoverContent = (
-    <Popover.Content id={confirmId}>
+    <Popover.Content id={confirmId} data-pka-anchor={popoverContentAnchor}>
```

Comment:
Being able to add this custom data anchor allows us to target the confirmation component button even if there are multiple confirmation or popover components open (for whatever reason) as the content is rendered in a portal at the base of the dom

Reply:
reverted this change now as there should only be one portal content rendered at any one time, so we should be able to target the default popover content qa selector